index.html

```diff
@@ -110,7 +110,9 @@
   local destdir=${DESTDIR:-~/.local/bin}
   local dest=${destdir}/subctl
 
-  install -D "${bin}" "${dest}"
+  mkdir -p "$destdir" && \
```

Comment:
`set -e` is active, so

```suggestion
  mkdir -p "$destdir"
  cp -f "${bin}" "${dest}"
  chmod +x "${dest}"
```

will work just as well.

Reply:
@SteveMattar sorry for the delay, do you mind updating the patch ^, I know it took so long to get attention.

Reply:
Done